### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by WPcdi** [Reza-Zhu/MBF#4]
Is any of the text in the dataset publicly available?

**Comment by Reza-Zhu** [comment]
No, I generate the texts and embeddings through the information given in the paper

**Comment by WPcdi** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，您的邮件已收到。

**Comment by WPcdi** [comment]
您好，请问论文里面的block与HBP的作用有什么区别吗，好像都是将transformer融合后的特征再和resnet特征融合一次?




爱哭爱笑的小妖精
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Reza-Zhu/MBF"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月5日(星期五) 中午11:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Reza-Zhu/MBF] dataset (Issue #4)





 
No, I generate the texts and embeddings through the information given in the paper
